app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    is_cp, command_to_read, command_to_write = command.split()
+    if is_cp != "cp":
+        raise ValueError("The command must start with the command 'cp'")
```

Comment:
No need to raise here - simply combine it with other condition in one if statement and execute code if both are satisfied. If not satisfied - do nothing